src/Data/NodeResolver.php

```diff
@@ -96,12 +96,26 @@ public function resolve_uri( string $uri, $extra_query_vars = '' ) {
 
 		$parsed_url = wp_parse_url( $uri );
 
+		if ( false === $parsed_url ) {
+			graphql_debug( __( 'Cannot parse provided URI', 'wp-graphql' ), [
```

Comment:
it's rare, but `wp_parse_url()` can sometimes fail to parse if it gets malformed input. This also prevents a handful of php errors from accessing array keys on an `array|false`